### PR TITLE
Update .travis.yml

### LINKED ISSUE #1552: Python 3.7 support
Currently, the Travis-CI configuration does not include Python 3.7 and should be added to [.travis.yml](https://github.com/exercism/python/blob/master/.travis.yml). As the default Linux distribution used by Travis (Trusty) is not supported for Python 3.7, see the following workaround: https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-402045581

### PR DESCRIPTION
**Pull request by spamyak**
Add support for Python 3.7. Fix #1552

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,14 +2,26 @@ sudo: false
...
+          sudo: false
+        - python: 3.6
+          dist: trusty
+          sudo: false
+        - python: 3.7
+          dist: xenial
+          sudo: true
+        - python: nightly
+          dist: trusty
+          sudo: false
```
Comment (cmccandless) [inline .travis.yml]: Indents should be done with 2 spaces, not 4. Please correct the indentation as Travis cannot parse mixed indentation.

**Comment by spamyak** [comment]
I've modified the spacing as requested. This should fix #1552 

Apologies for the poor quality of my initial contribution, I'm still learning to work with Github.

**Comment by cmccandless** [comment]
> This should fix #1552

The magic words need to be in the PR description. Please edit the description, and add "Fix #1552" to the end.

> Apologies for the poor quality of my initial contribution, I'm still learning to work with Github.

No worries! What's important is that you are here, contributing! Just make sure to always listen to feedback, but if you are confident that the feedback is incorrect, always back up your position with documentation.

**Comment by cmccandless** [comment]
Merged; thanks for working on this!

**Pull request merged**